### PR TITLE
Bug#1532704 - Added a note for enabling dynamic provisioning by using ansible variable

### PR DESCRIPTION
**Pull request by gaurav-nelson**
for https://bugzilla.redhat.com/show_bug.cgi?id=1532704

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@thom-at-redhat PTAL

**Comment by gaurav-nelson** [comment]
@thom-at-redhat I have tagged this PR for 3.7 and up. Is it applicable to OCP 3.6 as well?

**Comment by gaurav-nelson** [comment]
@openshift/team-documentation PTAL, waiting for QE review, but do not expect major changes 🔢

File: install_config/persistent_storage/dynamically_provisioning_pvs.adoc [hunk 1]
```diff
@@ -32,6 +32,22 @@ Many storage types are available for use as persistent volumes in
 administrator, some types of storage are created dynamically using the
 built-in provider and plug-in APIs.
 
+[NOTE]
+====
+To enable dynamic provisioning, declare the
+`openshift_master_dynamic_provisioning_enabled` variable (within the `[OSEv3:vars]` section) in the Ansible
+inventory file and set its value to `True`.
```
Comment (kalexand-rh) [inline install_config/persistent_storage/dynamically_provisioning_pvs.adoc]: I'd say "To enable dynamic provisioning, add the `openshift_master_dynamic_provisioning_enabled` variable to the `[OSEv3:vars]` section of the Ansible inventory file and set its value to `True`."

IBM Style doesn't like parentheses.

Comment (gaurav-nelson) [reply install_config/persistent_storage/dynamically_provisioning_pvs.adoc]: Thanks @kalexand-rh I'll fix this.

**Comment by jhou1** [comment]
LG

**Pull request merged**